ios/FluentUI/AvatarGroup/AvatarGroupTokens.swift

```diff
@@ -54,7 +54,9 @@ public class AvatarGroupTokens: ControlTokens {
     /// CGFloat that defines the thickness of the ring around the `Avatar`.
     open var ringThickness: CGFloat {
```

Comment:
Now that we have `AvatarTokens`, should these duplicate values be removed? I only see `interspace` in the legacy tokens.

Reply:
Right, interspace was the only token defined under AvatarGroup before, but it was just getting these values from an Avatar token it created. I think the change I'm working on to refactor the Avatar and Avatar Group will take care of removing these duplicate values.